hazelcast/src/main/java/com/hazelcast/scheduledexecutor/impl/ScheduledExecutorContainer.java

```diff
@@ -169,17 +159,12 @@ public void enqueueSuspended(TaskDefinition definition) {
 
     public void enqueueSuspended(ScheduledTaskDescriptor descriptor, boolean force) {
         if (logger.isFinestEnabled()) {
```

Comment:
Since you perform this check in the `log` method, you can basically remove all other `logger.is*Enabled` checks.

Reply:
I thought the same, even written review comments everywhere. But Thomas just kept them where he concats new strings as log parameters. He removed those guards when just constant strings were used. So I think this is okay.

Looking forward to Java 8, where we could pass a lambda, which will just be executed in the case of the logging. This would make all those guards unnecessary.

Reply:
Exactly, I only guarded where we have concats